scm-core/src/main/java/sonia/scm/repository/work/SimpleWorkingCopyFactory.java

```diff
@@ -100,9 +104,28 @@
   private static final Logger LOG = LoggerFactory.getLogger(SimpleWorkingCopyFactory.class);
 
   private final WorkingCopyPool workingCopyPool;
+  private final MeterRegistry meterRegistry;
 
+  /**
+   * Constructs a new {@link SimpleWorkingCopyFactory}
+   *
+   * @param workingCopyPool pool which provides working copies
+   * @deprecated since 2.16.0 use {@link SimpleWorkingCopyFactory(WorkingCopyPool, MeterRegistry)} instead
+   */
+  @Deprecated
   public SimpleWorkingCopyFactory(WorkingCopyPool workingCopyPool) {
+    this(workingCopyPool, new SimpleMeterRegistry());
```

Comment:
Why the `SimpleMeterRegistry`? Nobody can consume the collected metrics. A better solution would be an empty `CompositeMeterRegistry`.